packages/ui/src/fields/Join/index.tsx

```diff
@@ -59,9 +59,10 @@ const JoinFieldComponent: JoinFieldClientComponent = (props) => {
       className={[fieldBaseClass, 'join'].filter(Boolean).join(' ')}
       id={`field-${path?.replace(/\./g, '__')}`}
     >
-      {BeforeInput}
```

Comment:
I think we should keep the field's admin components in the Join component instead of passing through to RelationshipTable. Was there a reason you did it this way?

Reply:
@DanRibbens Yes - in the other fields we render the `beforeInput` below the name/label of the field.

In the previous version, the location of that beforeInput would render above the name/label

Reply:
I did this to keep consistency - the other option would be to render the label component out from the RelationshipTable component and directly into the Join field component - not sure about that one though.

Reply:
Oh yeah, makes sense. Too late to refactor these components now, so this is fine. We probably could have made these components more modular and reusable if we had split them up differently, oh well!